src/components/SliderWithInput.tsx

```diff
@@ -0,0 +1,128 @@
+import { css } from "@emotion/react";
+import {
+  Input,
+  InputProps,
+  SliderUnstyled,
+  SliderUnstyledProps,
+  Theme,
+} from "@mui/material";
+
+import React from "react";
+
+export type SliderWithInputProps = {
+  label: string;
+  inputProps?: InputProps;
+  sliderProps?: SliderUnstyledProps;
+};
```

Comment:
Two things--
1. We can just use `export interface` here (and probably should unless we're concatenating types with `&`)
2. Why not declare `SliderWithInput: React.VFC<SliderUnstyledProps>` instead of making another interface? We're basically presenting the API of a regular `SliderUnstyled` with some extra style atop it, right?

Reply:
1. So that's what that does. Kek
2. We can't use just SliderUnstyledProps, because SliderUnstyled doesn't include the input element we've added. It needs to take both a SliderUnstyledProps and an InputProps, which a normal SliderUnstyled doesn't have. It's not just extra style.